java/src/test/android/app/src/androidTest/java/ai/onnxruntime/extensions/apptest/BertTokenizerTest.java

```diff
@@ -21,18 +22,36 @@
 import ai.onnxruntime.OrtSession;
 import ai.onnxruntime.extensions.OrtxPackage;
 
+import org.junit.Rule;
+import org.junit.After;
+import com.microsoft.appcenter.espresso.Factory;
+import com.microsoft.appcenter.espresso.ReportHelper;
+
 /**
  * Instrumented test, which will execute on an Android device.
  *
  * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
  */
 @RunWith(AndroidJUnit4.class)
 public class BertTokenizerTest {
+
+    @Rule
```

Comment:
With this change, can we still run this unit test with gradle command line directly?

Reply:
Yes, the unit test with command `.\gradlew --no-daemon clean connectedDebugAndroidTest` still works.